probe/endpoint/ebpf.go

```diff
@@ -47,50 +40,97 @@ func NewEbpfTracker(bccProgramPath string) *EbpfTracker {
 
 	stderr, err := cmd.StderrPipe()
 	if err != nil {
-		log.Errorf("bcc error: %v", err)
+		log.Errorf("EbpfTracker error: %v", err)
 		return nil
 	}
-	go logPipe("bcc stderr:", stderr)
+	go logPipe("EbpfTracker stderr:", stderr)
 
 	tracker := &EbpfTracker{
-		Cmd: cmd,
+		cmd:         cmd,
+		activeFlows: map[string]ebpfConnection{},
 	}
 	go tracker.run()
 	return tracker
 }
 
+func (t *EbpfTracker) handleFlow(eventStr string, tuple fourTuple, pid int, netns string) {
+	t.Lock()
+	defer t.Unlock()
+
+	switch eventStr {
+	case "connect":
+		log.Infof("EbpfTracker: connect: %s pid=%v", tuple.String(), pid)
+		conn := ebpfConnection{
+			outgoing: true,
+			tuple:    tuple,
+			pid:      pid,
+			netns:    netns,
+		}
+		t.activeFlows[tuple.String()] = conn
+	case "accept":
+		log.Infof("EbpfTracker: accept: %s pid=%v", tuple.String(), pid)
+		conn := ebpfConnection{
+			outgoing: true,
```

Comment:
Not sure, it looks like this "outgoing" field could be deleted?

I initially thought it would be useful for the reporter to know when to reverse the tuple (see https://github.com/weaveworks/scope/blob/master/probe/endpoint/reporter.go#L177). But I didn't test whether it is needed.